.circleci/config.yml

```
@@ -24,6 +24,22 @@ rc_branch_only: &rc_branch_only
       only:
         - /^Version-v(\d+)[.](\d+)[.](\d+)/
 
+aliases:
+  # Shallow Git Clone
+  - &shallow-git-clone
+    name: Shallow Git Clone
+    command: |
+      #!/bin/sh
```

Nit: It seems worth breaking this out into a separate file, in `.circleci/scripts`.  That way it'd get linted as well (we run shellcheck against our bash scripts).

---

It'd be nice to move that key into a constant as well, that explains what it is and where it came from

---

Making a script in `.circleci/scripts` is the first thing I tried, until I realized that you can't make a script in a git repo that executes **before** you clone the repo!  I did move the key into a constant with an explanation though.

---

Haha oh, of course. Never thought of that.

---

I ran this through [shellcheck](https://www.shellcheck.net/) manually; no errors but one warning, about the `set -o pipefail` option being Bash-specific. This resolves the warning:

```suggestion
      #!/bin/bash
```